runtime/src/main/java/org/corfudb/runtime/collections/PersistentCorfuTable.java

```diff
@@ -63,6 +63,7 @@ public void clear() {
 
     @Override
     public void close() {
+        proxy.close();
         // Evict all versions of this Table from MVOCache
```

Comment:
Is this still needed? `proxy.close()` will invoke `getUnderlyingMVO().close()` which will also do this invalidation.